.github/workflows/test.yml

```diff
@@ -25,3 +25,13 @@ jobs:
         run: |
           git diff --compact-summary --exit-code || \
           (echo; echo "Unexpected difference in directories after code generation. Run 'go generate' command and commit."; exit 1)
+
+  unit:
```

Comment:
we had some UTs but we were not running them 😄

Reply:
Fixed now:
![image](https://github.com/thousandeyes/terraform-provider-thousandeyes/assets/85952626/a880a7e3-e754-4c35-b369-6262d9a98721)